src/parsers/babel-eslint.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import pkg from 'acorn-to-esprima/package.json';
+import loadAndExecute from './utils/loadAndExecute';
+
+const ID = 'acorn-to-esprima';
+const name = 'babel-eslint';
```

Comment:
Since people don't know what acorn-to-esprima is - the dropdown option will be called `babel-eslint` but the version is `acorn-to-esprima-1.0.2` rather than the wrong `babelEslint-1.0.2` if that makes sense.